cdap-ui/app/cdap/text/text-en.yaml

```diff
@@ -294,7 +294,8 @@ features:
       support: Support
     Sidebar:
       extension: "Extensions:"
-
+    NamespaceDropdown:
+      add-ns: "Add Namespace"
```

Comment:
refrain using `-` in i18n. It's not a good practice to have a lot of `-` as a keyname